src/ontology/omo-edit.owl

```diff
@@ -645,6 +660,54 @@ EquivalentTo: xsd:integer[&gt; 2151 , &lt;= 2300]
     
 
 
+    <!-- http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym -->
+
+    <owl:AnnotationProperty rdf:about="http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym">
```

Comment:
What is a `set of referents`?

Reply:
a referent is the thing to which the name/primary refers to. I just copied and pasted these defs in from the tickets, I am happy to revise.

Reply:
Very unintelligible IMO. I get it now. The class is the set, the referents are individuals - this is silly I think (@cmungall sorry ) :) can you think of something better?

Reply:
Still would like to see a more normal definition here..